packages/apollo-gateway/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the the appropriate changes within that release will be moved into the new section.
 
-* _Nothing yet! Stay tuned!_
+* Gateway cached DataSource bug [#3412](https://github.com/apollographql/apollo-server/pull/3412)
```

Comment:
Could we include a more detailed message about what the bug was and what version it was introduced?

Reply:
Our changelog entries are typically pretty short with a link back to the PR.

Reply:
You know what best practices are better than I do, but I would imagine for major bugs that could affect your infrastructure _just not running_, we would have a more detailed changelog entry to encourage users who depend on the behavior to upgrade. 

From the perspective of someone who reads these changelogs / release notes often, I certainly would expect a more detailed reporting of major bugs and major new features. For instance, I wouldn't think `Gateway cached DataSource bug` would be something that causes my entire managed infrastructure to fail when my underlying services change URLs, but instead I would assume it was a bug around caching not being efficient enough.